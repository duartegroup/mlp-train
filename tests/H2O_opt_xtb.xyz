3
Generated by autodE on: 2025-03-03. 
O    -0.00110   0.36310  -0.00000
H    -0.82500  -0.18190  -0.00000
H     0.82610  -0.18120   0.00000
