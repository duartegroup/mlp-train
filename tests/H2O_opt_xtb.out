      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.7.1 (edcfbbe) compiled by 'runner@Mac-1732539443871.local' on 2024-11-25

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * C. Bannwarth, E. Caldeweyher, S. Ehlert, A. Hansen, P. Pracht,
     J. Seibert, S. Spicher, S. Grimme, WIREs Comput. Mol. Sci., 2020, 11,
     e01493. DOI: 10.1002/wcms.1493
   
   for GFN2-xTB:
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   for GFN1-xTB:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   for GFN0-xTB:
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   for GFN-FF:
   * S. Spicher and S. Grimme, Angew. Chem. Int. Ed., 2020, 59, 15665-15673.
     DOI: 10.1002/anie.202004239
   
   for ALPB and GBSA implicit solvation:
   * S. Ehlert, M. Stahn, S. Spicher, S. Grimme, J. Chem. Theory Comput.,
     2021, 17, 4250-4261. DOI: 10.1021/acs.jctc.1c00471
   
   for ddCOSMO and CPCM-X implicit solvation:
   * M. Stahn, S. Ehlert, S. Grimme, J. Phys. Chem. A,
     2023, XX, XXXX-XXXX. DOI: 10.1021/acs.jpca.3c04382
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   * E. Caldeweyher, J.-M. Mewes, S. Ehlert and S. Grimme, Phys. Chem. Chem. Phys.
     2020, 22, 8499-8512. DOI: 10.1039/D0CP00502A
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   * J. Koopman and S. Grimme, ACS Omega 2019, 4, 12, 15120-15133.
     DOI: 10.1021/acsomega.9b02011
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   for SPH calculations refer to:
   * S. Spicher and S. Grimme, J. Chem. Theory Comput., 2021, 17, 1701-1714
     DOI: 10.1021/acs.jctc.0c01306
   
   for ONIOM refer to:
   * C. Plett, A. Katbashev, S. Ehlert, S. Grimme, M. Bursch,
     Phys. Chem. Chem. Phys., 2023, 25, 17860-17868. DOI: 10.1039/D3CP02178E
   
   for DIPRO refer to:
   * J. Kohn, N. Gildemeister, S. Grimme, D. Fazzi, A. Hansen,
     J. Chem. Phys., 2023, just accepted.
   
   for PTB refer to:
   * S. Grimme, M. Mueller, A. Hansen, J. Chem. Phys., 2023, 158, 124111.
     DOI: 10.1063/5.0137838
   
   with help from (in alphabetical order)
   P. Atkinson, C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher
   M. Checinski, S. Dohm, S. Ehlert, S. Ehrlich, I. Gerasimov, C. Hölzer
   A. Katbashev, J. Kohn, J. Koopman, C. Lavigne, S. Lehtola, F. März, M. Müller,
   F. Musil, H. Neugebauer, J. Pisarek, C. Plett, P. Pracht, F. Pultar,
   J. Seibert, P. Shushkov, S. Spicher, M. Stahn, M. Steiner, T. Strunk,
   J. Stückrath, T. Rose, and J. Unsleber
   
 * started run on 2025/03/03 at 17:10:01.014     
   ID    Z sym.   atoms
    1    8 O      1
    2    1 H      2, 3

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : /Users/zoezhang/miniconda3/envs/mlptrain-mace/bin/xtb H2O_opt_xtb.xyz --chrg 0 --uhf 0 --opt
          coordinate file            : H2O_opt_xtb.xyz
          omp threads                :                     4


           -------------------------------------------------
          |                 G F N 2 - x T B                 |
           -------------------------------------------------

        Reference                      10.1021/acs.jctc.8b01176
      * Hamiltonian:
        H0-scaling (s, p, d)           1.850000    2.230000    2.230000
        zeta-weighting                 0.500000
      * Dispersion:
        s8                             2.700000
        a1                             0.520000
        a2                             5.000000
        s9                             5.000000
      * Repulsion:
        kExp                           1.500000    1.000000
        rExp                           1.000000
      * Coulomb:
        alpha                          2.000000
        third order                    shell-resolved
        anisotropic                    true
        a3                             3.000000
        a5                             4.000000
        cn-shift                       1.200000
        cn-exp                         4.000000
        max-rad                        5.000000


          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   6          :
          :  # atomic orbitals                   6          :
          :  # shells                            4          :
          :  # electrons                         8          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          :  net charge                          0          :
          :  unpaired electrons                  0          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.0918409 -0.509184E+01  0.422E+00   13.16       0.0  T
   2     -5.0935887 -0.174779E-02  0.239E+00   12.83       1.0  T
   3     -5.0938642 -0.275537E-03  0.437E-01   12.59       1.0  T
   4     -5.0939341 -0.698631E-04  0.940E-02   12.75       1.0  T
   5     -5.0939372 -0.308745E-05  0.630E-02   12.70       1.0  T
   6     -5.0939398 -0.263550E-05  0.871E-04   12.72      66.3  T
   7     -5.0939398  0.182373E-09  0.830E-04   12.72      69.5  T
   8     -5.0939398 -0.461059E-09  0.303E-05   12.72    1905.6  T

   *** convergence criteria satisfied after 8 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6725448             -18.3009
         2        2.0000           -0.5688385             -15.4789
         3        2.0000           -0.5010668             -13.6347
         4        2.0000           -0.4419855             -12.0270 (HOMO)
         5                          0.0255051               0.6940 (LUMO)
         6                          0.2133879               5.8066
      -------------------------------------------------------------
                  HL-Gap            0.4674906 Eh           12.7211 eV
             Fermi-level           -0.2082402 Eh           -5.6665 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.004 sec
 SCC setup                      ...        0 min,  0.000 sec (  4.061%)
 Dispersion                     ...        0 min,  0.000 sec (  1.086%)
 classical contributions        ...        0 min,  0.000 sec (  6.548%)
 integral evaluation            ...        0 min,  0.000 sec (  1.759%)
 iterations                     ...        0 min,  0.003 sec ( 79.621%)
 molecular gradient             ...        0 min,  0.000 sec (  3.493%)
 printout                       ...        0 min,  0.000 sec (  3.160%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -5.068109158313 Eh    ::
         :: gradient norm              0.042411896910 Eh/a0 ::
         :: HOMO-LUMO gap             12.721065701201 eV    ::
         ::.................................................::
         :: SCC energy                -5.093939823536 Eh    ::
         :: -> isotropic ES            0.031864824239 Eh    ::
         :: -> anisotropic ES          0.000254698333 Eh    ::
         :: -> anisotropic XC         -0.000665657774 Eh    ::
         :: -> dispersion             -0.000141217407 Eh    ::
         :: repulsion energy           0.025830665223 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :: total charge              -0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 
      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :   optimization level            normal          :
          :   max. optcycles                   200          :
          :   ANC micro-cycles                  20          :
          :   degrees of freedom                 3          :
          :.................................................:
          :   RF solver                   davidson          :
          :   write xtbopt.log                true          :
          :   linear?                        false          :
          :   energy convergence         0.5000000E-05 Eh   :
          :   grad. convergence          0.1000000E-02 Eh/α :
          :   maxmium RF displ.          1.0000000          :
          :   Hlow (freq-cutoff)         0.1000000E-01      :
          :   Hmax (freq-cutoff)         5.0000000          :
          :   S6 in model hess.         20.0000000          :
          ...................................................

generating ANC from exact Hessian ...
Using Lindh-Hessian (1995)
 Shifting diagonal of input Hessian by    0.0000000000000000     
 Lowest  eigenvalues of input Hessian
    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000
    0.259923    1.036089    1.375302
 Highest eigenvalues
    0.000000    0.000000    0.000000    0.259923    1.036089    1.375302


........................................................................
.............................. CYCLE    1 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.0939398 -0.509394E+01  0.372E-06   12.72       0.0  T
   2     -5.0939398 -0.710543E-14  0.166E-06   12.72   34785.7  T
   3     -5.0939398 -0.888178E-15  0.580E-07   12.72   99577.4  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -5.0681092 Eh     change       -0.5861978E-12 Eh
   gradient norm :     0.0424121 Eh/α   predicted     0.0000000E+00 (-100.00%)
   displ. norm   :     0.0979559 α      lambda       -0.3396358E-02
   maximum displ.:     0.0920613 α      in ANC's #1, #2, #3, ...

........................................................................
.............................. CYCLE    2 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.1027025 -0.510270E+01  0.233E-01   13.94       0.0  T
   2     -5.1027029 -0.359598E-06  0.137E-01   13.94       1.0  T
   3     -5.1027031 -0.171526E-06  0.522E-03   13.95      11.1  T
   4     -5.1027031 -0.699993E-08  0.197E-03   13.95      29.4  T
   5     -5.1027031 -0.436672E-09  0.324E-04   13.95     178.1  T
   6     -5.1027031 -0.161302E-10  0.612E-05   13.95     943.7  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -5.0703228 Eh     change       -0.2213641E-02 Eh
   gradient norm :     0.0107582 Eh/α   predicted    -0.1714474E-02 ( -22.55%)
   displ. norm   :     0.0451056 α      lambda       -0.4263423E-03
   maximum displ.:     0.0440333 α      in ANC's #1, #2, #3, ...

........................................................................
.............................. CYCLE    3 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.1054967 -0.510550E+01  0.955E-02   14.43       0.0  T
   2     -5.1054968 -0.829314E-07  0.561E-02   14.44       1.0  T
   3     -5.1054968 -0.233479E-07  0.215E-03   14.44      26.8  T
   4     -5.1054968 -0.236435E-08  0.550E-04   14.44     104.9  T
   5     -5.1054968 -0.187441E-10  0.132E-04   14.44     438.6  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -5.0705338 Eh     change       -0.2109903E-03 Eh
   gradient norm :     0.0036052 Eh/α   predicted    -0.2136057E-03 (   1.24%)
   displ. norm   :     0.0047387 α      lambda       -0.1325366E-04
   maximum displ.:     0.0036120 α      in ANC's #1, #2, #3, ...

........................................................................
.............................. CYCLE    4 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.1051180 -0.510512E+01  0.177E-02   14.40       0.0  T
   2     -5.1051181 -0.878534E-07  0.927E-03   14.40       6.2  T
   3     -5.1051181 -0.136401E-07  0.265E-03   14.41      21.8  T
   4     -5.1051181 -0.128596E-08  0.408E-04   14.40     141.4  T
   5     -5.1051181 -0.295053E-11  0.848E-06   14.40    6811.8  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -5.0705429 Eh     change       -0.9125866E-05 Eh
   gradient norm :     0.0013040 Eh/α   predicted    -0.6627740E-05 ( -27.37%)
   displ. norm   :     0.0048879 α      lambda       -0.3078562E-05
   maximum displ.:     0.0041070 α      in ANC's #1, #2, #3, ...

........................................................................
.............................. CYCLE    5 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.1048094 -0.510481E+01  0.165E-02   14.38       0.0  T
   2     -5.1048094 -0.758755E-07  0.863E-03   14.38       6.7  T
   3     -5.1048094 -0.116367E-07  0.248E-03   14.38      23.3  T
   4     -5.1048094 -0.110532E-08  0.365E-04   14.38     158.4  T
   5     -5.1048094 -0.233502E-11  0.903E-06   14.38    6396.7  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -5.0705441 Eh     change       -0.1137826E-05 Eh
   gradient norm :     0.0006559 Eh/α   predicted    -0.1538325E-05 (  35.20%)
   displ. norm   :     0.0016700 α      lambda       -0.5294493E-06
   maximum displ.:     0.0014160 α      in ANC's #1, #2, #3, ...

   *** GEOMETRY OPTIMIZATION CONVERGED AFTER 5 ITERATIONS ***

------------------------------------------------------------------------
 total energy gain   :        -0.0024349 Eh       -1.5279 kcal/mol
 total RMSD          :         0.0858109 a0        0.0454 Å
 total power (kW/mol):        -1.2785632 (step) -953.7389 (real)
------------------------------------------------------------------------

 ANCopt (total)                0 d,  0 h,  0 min,  0.007 sec
 optimizer setup                ...        0 min,  0.000 sec (  1.818%)
 model hessian                  ...        0 min,  0.000 sec (  4.335%)
 ANC generation                 ...        0 min,  0.000 sec (  1.534%)
 coordinate transformation      ...        0 min,  0.000 sec (  6.369%)
 single point calculation       ...        0 min,  0.002 sec ( 24.967%)
 optimization log               ...        0 min,  0.000 sec (  3.777%)
 hessian update                 ...        0 min,  0.000 sec (  0.238%)
 rational function              ...        0 min,  0.003 sec ( 48.479%)

================
 final structure:
================
3
 xtb: 6.7.1 (edcfbbe)
O           -0.00030504471882        0.37999649088308       -0.00000000000000
H           -0.77174738205111       -0.19060485345029        0.00000000000000
H            0.77205242676993       -0.18939163743279       -0.00000000000000

 Bond Distances (Angstroems)
 ---------------------------
O1-H2=0.9595         O1-H3=0.9596         H2-O1=0.9595         H3-O1=0.9596         
 O  H  Rav=0.9595 sigma=0.0000  Rmin=0.9595  Rmax=0.9596     2

 selected bond angles (degree)
 --------------------
H3-O1-H2=107.11                

 selected dihedral angles (degree)
 ---------------------------------

           -------------------------------------------------
          |                Final Singlepoint                |
           -------------------------------------------------

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   6          :
          :  # atomic orbitals                   6          :
          :  # shells                            4          :
          :  # electrons                         8          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          :  net charge                          0          :
          :  unpaired electrons                  0          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.1048094 -0.510481E+01  0.480E-06   14.38       0.0  T
   2     -5.1048094  0.266454E-14  0.289E-06   14.38   19953.6  T
   3     -5.1048094 -0.888178E-15  0.721E-08   14.38  100000.0  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6800902             -18.5062
         2        2.0000           -0.5681995             -15.4615
         3        2.0000           -0.5110184             -13.9055
         4        2.0000           -0.4464134             -12.1475 (HOMO)
         5                          0.0821287               2.2348 (LUMO)
         6                          0.2504450               6.8150
      -------------------------------------------------------------
                  HL-Gap            0.5285421 Eh           14.3824 eV
             Fermi-level           -0.1821424 Eh           -4.9563 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.001 sec
 SCC setup                      ...        0 min,  0.000 sec ( 26.389%)
 Dispersion                     ...        0 min,  0.000 sec (  1.343%)
 classical contributions        ...        0 min,  0.000 sec (  0.972%)
 integral evaluation            ...        0 min,  0.000 sec (  7.593%)
 iterations                     ...        0 min,  0.000 sec ( 34.537%)
 molecular gradient             ...        0 min,  0.000 sec ( 10.093%)
 printout                       ...        0 min,  0.000 sec ( 17.083%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -5.070544053370 Eh    ::
         :: gradient norm              0.000655855279 Eh/a0 ::
         :: HOMO-LUMO gap             14.382362102332 eV    ::
         ::.................................................::
         :: SCC energy                -5.104809439869 Eh    ::
         :: -> isotropic ES            0.031455517002 Eh    ::
         :: -> anisotropic ES          0.000406062798 Eh    ::
         :: -> anisotropic XC         -0.000875092355 Eh    ::
         :: -> dispersion             -0.000141099538 Eh    ::
         :: repulsion energy           0.034265386500 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :: total charge              -0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6800902             -18.5062
         2        2.0000           -0.5681995             -15.4615
         3        2.0000           -0.5110184             -13.9055
         4        2.0000           -0.4464134             -12.1475 (HOMO)
         5                          0.0821287               2.2348 (LUMO)
         6                          0.2504450               6.8150
      -------------------------------------------------------------
                  HL-Gap            0.5285421 Eh           14.3824 eV
             Fermi-level           -0.1821424 Eh           -4.9563 eV

     #   Z          covCN         q      C6AA      α(0)
     1   8 O        1.611    -0.565    24.354     6.661
     2   1 H        0.805     0.282     0.777     1.384
     3   1 H        0.805     0.282     0.777     1.384

 Mol. C6AA /au·bohr⁶  :         44.534520
 Mol. C8AA /au·bohr⁸  :        795.707500
 Mol. α(0) /au        :          9.429114


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom

 ---------------------------------------------------------------------------
     #   Z sym  total        # sym  WBO       # sym  WBO       # sym  WBO
 ---------------------------------------------------------------------------
     1   8 O    1.840 --     3 H    0.920     2 H    0.920
     2   1 H    0.920 --     1 O    0.920
     3   1 H    0.920 --     1 O    0.920
 ---------------------------------------------------------------------------

Topologies differ in total number of bonds
Writing topology from bond orders to xtbtopo.mol

molecular dipole:
                 x           y           z       tot (Debye)
 q only:        0.000      -0.608       0.000
   full:        0.001      -0.873       0.000       2.219
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:        1.310       0.002      -0.819      -0.000       0.000      -0.491
  q+dip:        1.745       0.002      -1.175      -0.000       0.000      -0.570
   full:        1.950       0.002      -1.120       0.000       0.000      -0.829

           -------------------------------------------------
          |                Geometry Summary                 |
           -------------------------------------------------

      molecular mass/u    :       18.0152864
   center of mass at/Å    :       -0.0002538       0.3162150      -0.0000000
  moments of inertia/u·Å² :        0.5816607E+00   0.1201123E+01   0.1782783E+01
rotational constants/cm⁻¹ :        0.2898190E+02   0.1403490E+02   0.9455795E+01

 * 2 selected distances

     #   Z          #   Z                                           value/Å
     1   8 O        2   1 H                                       0.9595360 (min)
     1   8 O        3   1 H                                       0.9595515 (max)

 * 1 distinct bonds (by element types)

   Z      Z             #   av. dist./Å        max./Å        min./Å
   1 H    8 O           2     0.9595438     0.9595515     0.9595360


optimized geometry written to: xtbopt.xyz


           -------------------------------------------------
          | TOTAL ENERGY               -5.070544053370 Eh   |
          | GRADIENT NORM               0.000655855279 Eh/α |
          | HOMO-LUMO GAP              14.382362102332 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2025/03/03 at 17:10:01.051     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.033 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.086 sec
 * ratio c/w:     2.571 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.004 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.014 sec
 * ratio c/w:     3.401 speedup
 ANC optimizer:
 * wall-time:     0 d,  0 h,  0 min,  0.009 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.032 sec
 * ratio c/w:     3.382 speedup

